Tutorials/ASP.NET/Blazor.Client/README.MD

```diff
@@ -0,0 +1,89 @@
+## Overview
+
+This tutorial demonstrates how to create a client-side Blazor application that uses the DevExpress ORM Library (XPO) to access and manage data. In this tutorial, we use an [intermediate Web API service](../../WebApi) to communicate with the database.
```

Comment:
"This tutorial demonstrates how to" How can a tutorial demonstrate something? A demo or video demonstrates something. A tutorial contains instructions or information.

Reply:
Yes